presets/4.3/vtx/NanoHawk.txt

```diff
@@ -0,0 +1,43 @@
+#$ TITLE: Emax NanoHawk
+#$ FIRMWARE_VERSION: 4.2 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: defaults, reset, clear, vtx, nanohawk, emax
+#$ AUTHOR: Sjoer van der Ploeg
+#$ DESCRIPTION: Resets VTx settings and table back to 4.3 Defaults
```

Comment:
i think this should be removed

Reply:
I did indeed copy the defaults, I kept the defaults in there for the case someone just grabs this preset and wants his VTX to work. Assuming someone would look for his or her model, then apply it regardless of what was set instead of going back to defaults first.

Reply:
I'm an idiot, I included the defaults the right way now :)